### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

- [x] depends on: #4360

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/PathConnected.lean [hunk 1]
```diff
@@ -326,11 +324,8 @@ def trans (γ : Path x y) (γ' : Path y z) : Path x z where
     refine'
       (Continuous.if_le _ _ continuous_id continuous_const (by norm_num)).comp
         continuous_subtype_val <;>
-    -- TODO: the following are provable by `continuity` but it is too slow
-    -- porting note: the new `continuity` succeeds and it isn't too slow!
+    -- Porting note: used `continuity`
```
Comment (semorrison) [inline Mathlib/Topology/PathConnected.lean:327]:
```suggestion
```
I think just drop this, it's confusing without the context, and doesn't really need to be recorded.

File: Mathlib/Topology/PathConnected.lean [hunk 2]
```diff
@@ -683,8 +678,8 @@ theorem truncate_continuous_family {X : Type _} [TopologicalSpace X] {a b : X} (
 @[continuity]
 theorem truncate_const_continuous_family {X : Type _} [TopologicalSpace X] {a b : X} (γ : Path a b)
     (t : ℝ) : Continuous ↿(γ.truncate t) := by
+  -- Porting note: used `continuity`
```
Comment (semorrison) [inline Mathlib/Topology/PathConnected.lean:681]: Similarly here.

**Comment by semorrison** [comment]
bors d+

**Comment by ChrisHughes24** [comment]
This can be merged after it's fixed

bors d+

**Comment by Ruben-VandeVelde** [comment]
I'll let https://github.com/leanprover-community/mathlib4/pull/4360 go first in case it conflicts

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#4360
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#4360~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**